connector-drawer/src/main/resources/application.properties

```diff
@@ -1,9 +1,11 @@
+notifications.connector.http.disable-faulty-endpoints=false
 notifications.connector.kafka.incoming.group-id=notifications-connector-drawer
 notifications.connector.kafka.incoming.topic=${mp.messaging.tocamel.topic}
 notifications.connector.kafka.outgoing.topic=${mp.messaging.fromcamel.topic}
 notifications.connector.kafka.outgoing.drawer.topic=${mp.messaging.todrawer.topic}
 notifications.connector.name=drawer
 notifications.connector.redelivery.counter-name=camel.drawer.retry.counter
+notifications.connector.supported-connector-headers=${notifications.connector.name}
```

Comment:
shoudn't we add `notifications.connector.http.components=http`, because the only used http service is recipients-resolver, and we use simple http to reach it.